src/main/java/com/minecolonies/coremod/blocks/BlockHutTownHall.java

```diff
@@ -22,4 +31,24 @@ public String getName()
     {
         return "blockHutTownHall";
     }
+
+    @Override
+    public void onBlockPlacedBy(
+            @NotNull final World worldIn, @NotNull final BlockPos pos, final IBlockState state, final EntityLivingBase placer, final ItemStack stack)
+    {
+        if(worldIn.isRemote)
+        {
+            return;
+        }
+
+        if(placer.getActiveHand().equals(EnumHand.MAIN_HAND))
+        {
+            final Colony colony = ColonyManager.getClosestColony(worldIn, pos);
+            if (colony == null && placer instanceof EntityPlayer)
+            {
+                ColonyManager.createColony(worldIn, pos, (EntityPlayer) placer);
+            }
+        }
+        super.onBlockPlacedBy(worldIn, pos, state, placer, stack);
```

Comment:
This should be at the top, or called before the return in the isRemote check. A super method may way to do something on the client.